### ISSUE TITLE
multichannel audio does not match input mixture length

### ISSUE THREAD

**Issue by pseeth** [interactiveaudiolab/nussl#62]
When working with multichannel input, given to nussl like this:

```
def load_audio(audio_file):
    print 'Loading %s' % audio_file
    audio, sr = librosa.load(audio_file, sr=None, mono=False, offset = 30, duration = 30)
    audio_signal = nussl.AudioSignal(audio_data_array = audio, sample_rate = sr)
    return audio_signal

def repet_sim(audio_signal):
    repet = nussl.RepetSim(audio_signal)
    repet.stft_params.hop_length = 1024
    repet.stft_params.n_fft_bins = 4096
    repet.stft_params.window_length = 4096
    repet.run()
    fg, bg = repet.make_audio_signals()
    return [bg.audio_data[0, :], fg.audio_data[0, :]]

audio_signal = load_audio(some_audio_file)
bg, fg = repet_sim(audio_signal)
```

I get the following error:
`ValueError: operands could not be broadcast together with shapes (2,1323008) (2,1323000) (2,1323008) `

Works fine when mono = True in the load_audio function.

**Comment by ethman** [comment]
Can you post the whole stack trace and an audio file?

**Comment by pseeth** [comment]
Fixed in aac8d69611a269e68e00a478fa5611dfe06f4a95 in development branch. Needs a merge.
